### PR TITLE
Fix Tracking Seed and Candidate Plots, add DCA and SIP to global Muons

### PR DESCRIPTION
**Pull request by VinInn**
Fix Seed and Candidate eta-phi plots: now using position not direction
Add vertex track-multiplicity vs z plot
Add DCA and SIP plots for globalMuons

### CODE REVIEW

**Comment by deguio** [comment]
please test

File: DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc [hunk 1]
```diff
@@ -296,9 +298,9 @@ void TrackBuildingAnalyzer::analyze
   GlobalPoint  v(v0.x()-bs.x0(),v0.y()-bs.y0(),v0.z()-bs.z0());
   
   double pt           = sqrt(state.globalMomentum().perp2());
-  double eta          = state.globalMomentum().eta();
-  double phi          = state.globalMomentum().phi();
-  double theta        = state.globalMomentum().theta();
+  double eta          = state.globalPosition().eta();
```
Comment (slava77) [inline DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc]: while it's not in reco code, this looks odd to me.
I just wanted to check that these should be the detector surface eta/phi/theta values.
... same question for inner/outer position eta,phi (the innermost state position eta, say, for r_hit~4cm will be anywhere from ~-3 to +3  depending where the z_POCA is).

File: RecoTracker/IterativeTracking/python/TobTecStep_cff.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
 tobTecStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
     layerList = cms.vstring(
     #TOB
-    'TOB1+TOB2+MTOB3',
+    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
```
Comment (slava77) [inline RecoTracker/IterativeTracking/python/TobTecStep_cff.py]: remind me, please what "M" is for in MTOB4?

what's the effect of this on timing, efficiency and fake rates?
If there is any, please decouple changes in DQM plots from physics changes.

File: DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc (hunk 1 above)
Comment (VinInn) [inline DQM/TrackingMonitor/src/TrackBuildingAnalyzer.cc]: POCA is the main issue. we whant to correrate with detector issues, not to physics...
so indeed eta computed from direction was pretty random smearing any real feature in particular at large r.
now detector issue shows up immediately. will show examples tomorrow at trk dpg meeting

File: RecoTracker/IterativeTracking/python/TobTecStep_cff.py (hunk 2 above)
Comment (VinInn) [inline RecoTracker/IterativeTracking/python/TobTecStep_cff.py]: how it when in here?
I stashed before commit!
will remove!
still quite disappointed by git behaviour...

Comment (slava77) [inline RecoTracker/IterativeTracking/python/TobTecStep_cff.py]: On 9/24/15 2:11 PM, Vincenzo Innocente wrote:
> In RecoTracker/IterativeTracking/python/TobTecStep_cff.py
> <https://github.com/cms-sw/cmssw/pull/11467#discussion_r40359813>:
> 
>> @@ -21,7 +21,7 @@
>>  tobTecStepSeedLayersTripl = cms.EDProducer("SeedingLayersEDProducer",
>>      layerList = cms.vstring(
>>      #TOB
>> -    'TOB1+TOB2+MTOB3',
>> +    'TOB1+TOB2+MTOB3','TOB1+TOB2+MTOB4',
> 
> how it when in here?
> I stashed before commit!
> will remove!
> still quite disappointed by git behaviour...
> 

ok removal from a typo or something is an easy solution for me : the
reco  signature need goes away.

**Comment by VinInn** [comment]
will eventually fix HI

**Comment by VinInn** [comment]
140.53_RunHI2011+RunHI2011+RECOHID11+HARVESTDHI Step0-PASSED Step1-PASSED Step2-PASSED  - time date Sun Sep 27 14:49:04 2015-date Sun Sep 27 14:43:27 2015; exit: 0 0 0
1 1 1 tests passed, 0 0 0 failed

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**